DocSum/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -257,11 +256,21 @@ curl http://${HOST_IP}:${DOCSUM_BACKEND_SERVER_PORT}/v1/docsum \
    -F "max_tokens=32" \
    -F "language=en" \
    -F "stream=True"
+
+# Upload file
+curl http://${HOST_IP}:${DOCSUM_BACKEND_SERVER_PORT}/v1/docsum \
+   -H "Content-Type: multipart/form-data" \
+   -F "type=audio" \
+   -F "messages=" \
+   -F "files=@/path to your file (.mp3, .wav)" \
```

Comment:
Based on the code is looks like files is a list. Can multiple files be passed at the same time? If so, what would the curl command look like for that.

Reply:
I will add a note to the READMEs.